content/docs/addons-shallow-renderer.md

```diff
@@ -6,31 +6,31 @@ layout: docs
 category: Reference
 ---
 
-**Importing**
+**Mengimpor**
```

Comment:
```suggestion
**Cara Import**
```

Reply:
thank you for the suggestion and it's reference 👍